opennms-doc/guide-install/src/asciidoc/text/opennms/debian.adoc

```diff
@@ -3,48 +3,24 @@
 :imagesdir: ../../images
 
 [[gi-install-opennms-debian]]
-=== Install on Debian-based systems
+=== Debian
 
 IMPORTANT: This guide does not apply to OpenNMS Meridian, which can be installed only on Red Hat Enterprise Linux or CentOS systems.
 
 This section describes how to install the _OpenNMS_ platform on _Ubuntu 14.04 LTS_.
 The setup process is described in the following steps:
 
-. Install _OpenNMS_ apt repository server with GPG key to verify packages
 . Installation of the _opennms_ meta package which handles all dependencies
 . Initialize _PostgreSQL_ database and configure access
 . Initialize _OpenNMS_ and first start of the application
 
```

Comment:
Haven't found where do we describe how to setup the apt repository? Is this not necessary anymore?